next-app/src/routes/__layout.svelte

```diff
@@ -20,29 +20,29 @@
 	}
 </script>
 
-<svelte:window on:keydown={closeOnEscape} />
+<svelte:window on:keydown={ closeOnEscape } />
```

Comment:
I've noticed that some files, like this one, contain just whitespace changes.  The changes are great, and I assume that you have just run prettier or some other code formatter on the file.  I lament that we don't yet have a consistent code formatting solution that all devs are using.  I know it's not hard - just finding it hard to settle on Prettier, get all our code processed, and then enforce it at the commit level.

Reply:
sure, we can incorporate something like that if you'd like.  If it bothers you on code reviews, simply use the option available in GH to ignore whitespace changes during the review.  I can show you where to find that if you don't see it.

Reply:
yes, that's true about the whitespace changes.  I can use that filter.

I definitely want us to all be using a code formatter.